src/stream.ts

```diff
@@ -361,7 +361,7 @@ export class DataAndMoneyStream extends Duplex {
    */
   _addToIncoming (amount: BigNumber): void {
     this._totalReceived = this._totalReceived.plus(amount)
-    this.debug(`received ${amount} (totalReceived: ${this._totalReceived})`)
+    this.log.trace(`received ${amount} (totalReceived: ${this._totalReceived})`)
```

Comment:
on the fence if this should be debug, its chatty but useful for debugging purposes.